### ISSUE TITLE
Sega Model 2 and 3 Emulation on Series S/X (60$)

### ISSUE THREAD

**Issue by ArtytheStu** [Xbox-Homebrew/bounties#3]
I'm willing to pay 60$ right here and now to whoever pulls this off. Both Model 2 and 3 are emulated on PC and both should be open source.

I'd need them to run on Xbox, with working controller support and UI. There's so many great games Sega don't care about on these platforms, it's an absolute shame to let em rot like this.

Link to the bounty: https://www.bountysource.com/issues/98198907-sega-model-2-and-3-emulation-on-series-s-x-60

**Comment by tunip3** [comment]
link to bounty?

**Comment by ArtytheStu** [comment]
> link to bounty?

I don't know how it works exactly, how can I do this properly?

**Comment by ArtytheStu** [comment]
> link to bounty?

Ok took a bit but I figured out how it worked, here's the link to the bounty

https://www.bountysource.com/issues/98198907-sega-model-2-and-3-emulation-on-series-s-x-60

I'm editing the main post with the link as well

**Comment by JustMeDaFaq** [comment]
> I'm willing to pay 60$ right here and now to whoever pulls this off. Both Model 2 and 3 are emulated on PC and both should be open source.
> 
> I'd need them to run on Xbox, with working controller support and UI, and for them to be available on RETAIL since I have no dev mode available. There's so many great games Sega don't care about on these platforms, it's an absolute shame to let em rot like this.
> 
> Link to the bounty: https://www.bountysource.com/issues/98198907-sega-model-2-and-3-emulation-on-series-s-x-60

Availability on retail isnt something that should be up to an devleoper and probably even scares them away. Microsoft isnt welcoming emulators, they dont want them on the store. Once something comes up, either use dev mode (which is the proper way) or find someone who hosts it then for you, putting retail in a bounty as a requirement wont work out. @ArtytheStu

**Comment by ArtytheStu** [comment]
You're right, I didn't think about it. I'm going to remove the requirement immediately.

**Comment by lilo-san** [comment]
My understanding is that both emulators are Windows only and not work on from a long time ago:

Model 3
Emu Page: https://www.supermodel3.com
Source: https://sourceforge.net/p/model3emu/code/HEAD/tree/
Status: http://emulation.gametechwiki.com/index.php/Sega_Model_3

Model 2
Emu Page: http://nebula.emulatronia.com
Source: Private
Status: http://emulation.gametechwiki.com/index.php/Sega_Model_2

I'm surprised there is no more effort to improve support for these into MAME or any other emulator supported by Libretro so is easily ported to all consoles.

**Comment by danprice142** [comment]
@ArtytheStu Sega Model 3 is now a thing on Xbox https://github.com/aerisarn/Supermodel-uwp-gl
